src/job/jobHandler.ts

```diff
@@ -414,6 +418,50 @@ export abstract class JobHandler {
     }
   }
 
+  // Creates and pushes the related manifestJob
+  @throwIfJobInterupted()
+  public async queueManifestJob(): Promise<void> {
+    // Rudimentary error prevention
+    if (this._currJob.payload.jobType.includes('manifestGeneration')) {
```

Comment:
if i understand it right, we are not allowed to queue multiple manifest generation for same job?

Reply:
Yes, that's the idea - does that seem incorrect?